Gems/OpenXRVk/Code/Source/OpenXRVkInstance.cpp

```diff
@@ -11,6 +11,10 @@
 #include <Atom/RHI.Reflect/Vulkan/XRVkDescriptors.h>
 #include <AzCore/Casting/numeric_cast.h>
 
+#ifdef AZ_PLATFORM_ANDROID
+#include <AzCore/Android/AndroidEnv.h>
+#endif
+
```

Comment:
This file will use PAL in a follow up PR to avoid having to use android platform macro.

Reply:
This has been one, this is the PR: https://github.com/o3de/o3de-extras/pull/16